README.md

```diff
@@ -30,3 +30,5 @@ See also the list of [contributors](https://github.com/vatz88/FFCSonTheGo/contri
 ## License
 
 This project is licensed under the GNU General Public License - see the [LICENSE](LICENSE.md) file for details.
+
+"AAPUN NE BHI KAAM KIYA ISPAR"

```

Comment:
Hey @nimishjn thanks for your efforts in contributing but let's keep it civil and not add vage commits

Reply:
You could just be doing it for learning purpose, I'll close this PR and you can keep committing on your fork.